### PR TITLE
Remove unify ccc settings flag

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

**Review by yksolanki9 — CHANGES_REQUESTED** [review]
One comment. Also aren't we removing corporate card page and corresponding code from the sidemenu?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.html [hunk 1]
```diff
@@ -338,14 +336,14 @@
...
-            *ngIf="isUnifyCcceExpensesSettingsEnabled && !isCccExpense"
+            *ngIf="!isCccExpense"
             [type]="'information'"
             [message]="'Fyle will automatically merge this expense with the corresponding corporate card transaction when it flows into fyle.'"
           ></app-fy-alert-info>
         </ng-container>
 
         <ng-container *ngIf="isCCCPaymentModeSelected$|async">
-          <div *ngIf="showCardTransaction && (isCCCAccountSelected$|async) && !isUnifyCcceExpensesSettingsEnabled">
+          <div *ngIf="showCardTransaction && (isCCCAccountSelected$|async)">
```
[the remark this answers is not in the document]
Comment (hlkavya0213) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.html:346]: we should be removing the pages

[the remark this answers is not in the document]
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.html:346]: @hlkavya0213 the removal of pages will be taken care in a separate PR, I've mentioned it in the slack thread

[the remark this answers is not in the document]
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.html:346]: @yksolanki9 yes, this block will be removed

[the remark this answers is not in the document]
Comment (hlkavya0213) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.html:346]: need to remove whole section

**Review by hlkavya0213 — APPROVED** [review]

**Pull request closed without merging**